docs/notebooks/DESC_Solve_from_pyQSC.ipynb

```diff
@@ -0,0 +1,250 @@
+   "metadata": {},
+   "source": [
+    "# Creating a DESC Equilibrium from a pyQsc Near-Axis Equilibrium"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a6ee32b-1a4c-47f5-9a9d-3498fb7d3f55",
+   "metadata": {},
+   "source": [
+    "Note that you must have pyQsc installed in order to make use of the `Equilibrium.from_near_axis` method, do so with `pip install qsc`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "a1f1791e-97bf-4a56-8968-4650d48f003d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DESC version 0.7.2+319.g66640f23.dirty,using JAX backend, jax version=0.2.25, jaxlib version=0.1.76, dtype=float64\n",
+      "Using device: CPU, with 21.62 GB available memory\n"
+     ]
+    }
+   ],
+   "source": [
+    "# must have installed pyQsc with `pip install qsc` in order to use this!\n",
+    "from qsc import Qsc\n",
+    "from desc.equilibrium import Equilibrium\n",
+    "from desc.objectives import get_fixed_boundary_constraints\n",
+    "from desc.plotting import plot_comparison"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "90a66694-ccee-43e5-909c-927681091871",
+   "metadata": {},
+   "source": [
+    "DESC is able to create an equilibrium based off of a `pyQsc` NAE equilibrium object. First, we'll make the NAE equilibrium using `pyQsc`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0403611d-9bd7-44cf-88ff-9c848ba3c1d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qsc_eq = Qsc.from_paper(\"precise QA\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed61eac3-54f8-49a2-bb46-23e73e07ea06",
+   "metadata": {},
+   "source": [
+    "Then, to make the DESC equilibrium, the `Equilibrium` class has a method `Equilibrium.from_near_axis`. This method creates a DESC `Equilibrium` based off of the `pyQsc` equilibrium. It requires as input the desired DESC FourierZernike resolution, as well as the radius at which you want to evaluate the qsc equilibrium at to make the DESC equilibrium's boundary. The equilibrium's initial `R_lmn`, `Z_lmn` Fourier-Zernike coefficients are fit to the `R,Z` evaluated from the `pyQsc` equilibrium, and the initial `L_lmn` are 0 (because the `pyQsc` equilibrium uses Boozer angles, so there is no poloidal stream function)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "d490a021-dc21-421b-b985-16dcaf832974",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ntheta = 75\n",
+    "r = 0.15\n",
+    "desc_eq = Equilibrium.from_near_axis(\n",
+    "    qsc_eq,  # the Qsc equilibrium object\n",
+    "    r=r,  # the finite radius (m) at which to evaluate the Qsc surface to use as the DESC boundary\n",
+    "    L=8,  # DESC radial resolution\n",
+    "    M=8,  # DESC poloidal resolution\n",
+    "    N=8,  # DESC toroidal resolution\n",
+    "    ntheta=ntheta,\n",
```

Comment:
suggest to add the comment:  `# number of poloidal grid points used in the conversion`